tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -519,20 +519,23 @@ def _test_bed(self):
             logger.info('Topology config devices for case %s: %s', case_id, topo_mixed_devices)
             f_topo.close()
             needed_golden_devices = []
```

Comment:
rename `needed_golden_devices` to `existing_golden_devices` or `founded_golden_devices`?